src/schema/types/attribution_class.js

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
We're a bit inconsistent on where we put these, would you mind moving to `src/schema/artwork` folder? That way attribution and all those other 'partials' are colocated with the `index` which uses all of them.